src/bundle/Resources/public/scss/_general.scss

```diff
@@ -145,6 +146,25 @@ select {
     background-color: $ibexa-color-base-pale;
 }
 
+.ez-info-view,
```

Comment:
Personally I would add new single class to css and add this class to html instead of styling by all these 20 classes